inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/internal/JdbcMultiBatchingComm.java

```diff
@@ -173,4 +174,47 @@ private static RowData getPrimaryKey(RowData row, RowData.FieldGetter[] fieldGet
         return pkRow;
     }
 
+    public static JdbcOptions getExecJdbcOptions(JdbcOptions jdbcOptions, String tableIdentifier) {
+        JdbcOptions jdbcExecOptions =
+                JdbcOptions.builder()
+                        .setDBUrl(jdbcOptions.getDbURL() + "/" + getDatabaseNameFromIdentifier(tableIdentifier))
+                        .setTableName(getTableNameFromIdentifier(tableIdentifier))
+                        .setDialect(jdbcOptions.getDialect())
+                        .setParallelism(jdbcOptions.getParallelism())
+                        .setConnectionCheckTimeoutSeconds(jdbcOptions.getConnectionCheckTimeoutSeconds())
+                        .setDriverName(jdbcOptions.getDriverName())
+                        .setUsername(jdbcOptions.getUsername().orElse(""))
+                        .setPassword(jdbcOptions.getPassword().orElse(""))
+                        .build();
+        return jdbcExecOptions;
+    }
+
+    /**
+     * Get table name From tableIdentifier
+     * tableIdentifier maybe: ${dbName}.${tbName} or ${dbName}.${schemaName}.${tbName}
+     *
+     * @param tableIdentifier The table identifier for which to get table name.
+     */
+    public static String getTableNameFromIdentifier(String tableIdentifier) {
+        String[] fileArray = tableIdentifier.split("\\.");
```

Comment:
please use a name that has the right meaning

Reply:
Maybe IdentifierArray?

Reply:
fieldArray done